test/kennel/optional_validations_test.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 require_relative "../test_helper"
 
-SingleCov.covered!
+SingleCov.covered! uncovered: 19
```

Comment:
I got lazy. Most of this code goes away in the next PR. Let me know if that's not good enough.